modules/storage-expanding-csi-volumes.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies
+//
+// * storage/expanding-persistent-volumes.adoc
+
+[id="expanding-csi-volumes_{context}"]
+= Expanding CSI volumes
+
+Support for expanding CSI volumes is enabled by default, but it also requires a specific CSI driver to support volume expansion.
```

Comment:
There's a lot of passive voice in this module. It's a necessary evil when you're doing a lot of writing about system processes, but do you think it's possible to cut down on it with some revision?

Reply:
Gave it another pass (sorry for the pun)